ribs/archives/_cvt_archive.py

```diff
@@ -260,12 +264,30 @@ def index_of(self, measures_batch):
             _, indices = self._centroid_kd_tree.query(measures_batch)
             return indices.astype(np.int32)
 
+        expanded_dims = np.expand_dims(measures_batch, axis=1)
+        if self.chunk_distances:  # Compute indices chunks at a time
+            distances = np.sum(np.square(
+                expanded_dims[0:min(len(measures_batch), self.chunk_size)] -
+                self.centroids),
+                               axis=2)
+            indices = np.argmin(distances, axis=1).astype(np.int32)
+            for i in range(
+                    2,
+                    len(measures_batch) // self.chunk_size +
```

Comment:
I feel like there's a less verbose way to represent this -- maybe a ceiling operation?

Reply:
Yes that should work. Can you locally print out the indices that are being used in this loop (i.e., `l` and `r`) and check that they're what you expect?

Reply:
Just checked, they're as expected